components/metris/.dockerignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
Please also modify CODEOWNERS file (on root) to include you and other component owners there

Reply:
Please also add chart - it may be done in separate PR of course. But in the same PR you can include chart path in CODEOWNERS as well